src/docs/getting_started/advanced_builder.md

```diff
@@ -0,0 +1,37 @@
+# Advanced Builder Concepts
+
+There are a number of features in `maggma` designed to assist with advanced features:
+
```

Comment:
Maybe a list of advanced builders here also? With the MapBuilder one linking to the nice MapBuilder doc.